styleguide/source/_patterns/01-molecules/09-search-results/_search-results.scss

```diff
@@ -0,0 +1,207 @@
+.search-result {
+  margin-bottom: $space;
+  width: 100%;
+  display: flex;
+  cursor: pointer;
+
+  &:active,
+  &:hover {
+    color: $blue;
+
+    .search-result_link {
+      text-decoration: underline;
+    }
+
+    .search-result_left {
+      background-color: $black-15;
+    }
+
+    .search-result_label {
+      color: $blue;
+    }
+
+    .external {
+      fill: $blue;
+    }
+  }
+}
+
+.search-result_label {
```

Comment:
So..in one of my PRs I made a label parargraph (PR is not merged yet..but should be merged soon) that I think we can reuse in place of this more specific label.